packages/core/utils/package.json

```diff
@@ -8,5 +8,8 @@
   },
   "author": "Fluffy Labs",
   "license": "MPL-2.0",
-  "type": "module"
+  "type": "module",
+  "dependencies": {
+    "@typeberry/logger": "*"
+  }
```

Comment:
I think this causes a circular dependency. I'd rather keep `utils` without logger dep. Feel free to user `console` directly and ignore linter on that line.

Reply:
fixed!